TypeDBConsole.java

```diff
@@ -839,33 +782,15 @@ public void run() {
         }
 
         private void validateOptions() {
-            if (core != null && cloud != null) {
+            if (core != null && cloud != null)
                 throw new CommandLine.ParameterException(spec.commandLine(), "Either '--core' or '--cloud' must be provided, but not both.");
-            } else {
-                if (cloud != null) validateCloudOptions();
-                else validateServerOptions();
-            }
-        }
-
-        private void validateServerOptions() {
-            if (username != null)
-                throw new CommandLine.ParameterException(spec.commandLine(), "'--username' should only be supplied with '--cloud'");
-            if (password != null)
-                throw new CommandLine.ParameterException(spec.commandLine(), "'--password' should only be supplied with '--cloud'");
-            if (tlsEnabled)
-                throw new CommandLine.ParameterException(spec.commandLine(), "'--tls-enabled' is only valid with '--cloud'");
-            if (tlsRootCA != null)
-                throw new CommandLine.ParameterException(spec.commandLine(), "'--tls-root-ca' is only valid with '--cloud'");
+            if (cloud != null) validateCloudOptions();
+            if (!tlsEnabled && tlsRootCA != null)
+                throw new CommandLine.ParameterException(spec.commandLine(), "'--tls-root-ca' should only be supplied when '--tls-enabled' is set to 'true'");
```

Comment:
Since `username` and `password` are now marked as `required` on by picocli, we don't need to check for them being non-null. Additionally, we move the checks for TLS parameters into the shared `validateOptions` as they can now be used by both core and cloud.

Since there's now no core-unique validation required, we can dissolve `validateServerOptions` entirely.